controllers/nodeupgrade_controller.go

```diff
@@ -168,14 +166,20 @@ func (r *NodeUpgradeReconciler) reconcile(ctx context.Context, log logr.Logger,
 		return ctrl.Result{}, nil
 	}
 
+	configMap := &corev1.ConfigMap{}
+	if err := remoteClient.Get(ctx, types.NamespacedName{Name: constants.UpgraderConfigMapName, Namespace: constants.EksaSystemNamespace}, configMap); err != nil {
```

Comment:
So we are planning to retrieve the config map and pass the image directly to NodeUpgrader object? What's stopping us from doing this Get operation in the nodeupgrade controller then?

Other option is so pass in the config map from here to node upgrade object, just so that if node upgrader is run on it's own for any reason, we can pass in any config map. If we decide to mount the config map to the upgrader pod directly, then that gives us a different amount of flexibility where we can deploy the pod directly with a different config map mounted. Just some thoughts that I have to consider.